tasks/setup.yaml

```diff
@@ -51,9 +51,15 @@ tasks:
 
   - name: create-doug-user
     description: Creates a user named 'doug' in the uds realm of keycloak (using the default admin account)
+    inputs:
+      keycloak_group:
+        description: Group to add user to
+        default: ""
```

Comment:
```suggestion
        default: "$KEYCLOAK_GROUP"
```

Reply:
we should maintain variable compatibility - as is this will always ignore the variable at the top

Reply:
nice catch!